sdk/identity/azure-identity/azure/identity/_credentials/chained.py

```diff
@@ -37,7 +37,8 @@ class ChainedTokenCredential:
     """A sequence of credentials that is itself a credential.
 
     Its :func:`get_token` method calls ``get_token`` on each credential in the sequence, in order, returning the first
-    valid token received.
+    valid token received. For more information, see
+    https://aka.ms/azsdk/python/identity/credential-chains#chainedtokencredential-overview.
```

Comment:
Can you also mirror this docstring update in the [async counterpart](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/azure/identity/aio/_credentials/chained.py)?